vertx-web-session-stores/vertx-web-sstore-caffeine/src/main/java/io/vertx/ext/web/sstore/caffeine/CaffeineSessionStore.java

```diff
@@ -0,0 +1,50 @@
+package io.vertx.ext.web.sstore.caffeine;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.sstore.SessionStore;
+import io.vertx.ext.web.sstore.caffeine.impl.CaffeineSessionStoreImpl;
+
+/**
+ * A session store based on the Caffeine which is only available on a single node.
+ * <p>
+ * Can be used when sticky sessions are being used.
```

Comment:
Perhaps could you add a note saying that sticky sessions is a third-party proxy feature, and that the session in this case won't survive a node crash.